node-fs-writefile/note-to-self-promise.js

```diff
@@ -0,0 +1,7 @@
+import { writeFile } from 'node:fs/promises';
+
+const data = `${process.argv[2]}\n`;
+const promise = writeFile('note.txt', data);
+promise
+  .then((value) => value)
```

Comment:
This doesn't do anything. Promises aren't _required_ to have `.then` handlers, so you could just leave this off.